accelerator/proxy_apis.c

```diff
@@ -60,6 +60,39 @@ status_t char_buffer_to_str(char_buffer_t* res_buff, char** json_result) {
   return SC_OK;
 }
 
+status_t api_proxy_apis(const iota_client_service_t* const service, const char* const obj, char** json_result) {
+  status_t ret = SC_OK;
+  const uint8_t max_cmd_len = 30;
+  char command[max_cmd_len];
+
+  proxy_apis_command_req_deserialize(obj, command);
+  // TODO more proxy APIs should be implemented
+  if (strncmp(command, "checkConsistency", strlen("checkConsistency")) == 0) {
```

Comment:
How about using a table with commands and corresponding callback functions. It can prevent more and more if-else as more APIs are implemented. Use a function to get the target function. We can iterate through the table or using data structures like Trie or others to improve efficency if necessary. (preventing do strncmp NUM_OF_APIs times)

Reply:
After discussing this issue. Use a table won't have a great influence at this moment.